integration-tests/src/test/groovy/com/okta/sdk/tests/it/PolicyRulesIT.groovy

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2018-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.client.Client
+import com.okta.sdk.resource.policy.GroupCondition
+import com.okta.sdk.resource.policy.OktaSignOnPolicy
+import com.okta.sdk.resource.policy.OktaSignOnPolicyRule
+import com.okta.sdk.resource.policy.OktaSignOnPolicyRuleActions
+import com.okta.sdk.resource.policy.OktaSignOnPolicyRuleConditions
+import com.okta.sdk.resource.policy.OktaSignOnPolicyRuleSignonActions
+import com.okta.sdk.resource.policy.OktaSignOnPolicyRuleSignonSessionActions
+import com.okta.sdk.resource.policy.PasswordPolicy
+import com.okta.sdk.resource.policy.PasswordPolicyRule
+import com.okta.sdk.resource.policy.PasswordPolicyRuleAction
+import com.okta.sdk.resource.policy.PasswordPolicyRuleActions
+import com.okta.sdk.resource.policy.PasswordPolicyRuleConditions
+import com.okta.sdk.resource.policy.Policy
+import com.okta.sdk.resource.policy.PolicyNetworkCondition
+import com.okta.sdk.resource.policy.PolicyPeopleCondition
+import com.okta.sdk.resource.policy.PolicyRule
+import com.okta.sdk.resource.policy.PolicyRuleAuthContextCondition
+import com.okta.sdk.resource.policy.UserCondition
+import org.testng.annotations.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+class PolicyRulesIT implements CrudTestSupport {
+
+    private OktaSignOnPolicy crudTestPolicy
+
+    @Override
+    def create(Client client) {
+
+        crudTestPolicy = client.createPolicy(client.instantiate(OktaSignOnPolicy)
+            .setName("policy+" + UUID.randomUUID().toString())
```

Comment:
We agreed some time ago to add an SDK identifier to the resources to trace possible issues from the Okta Org.
Is it still valid @vijetmahabaleshwar-okta ?

Reply:
@laura-rodriguez @vijetmahabaleshwar-okta lets define this separately and I can find/replace these all (along with groups, apps, factors, etc)